docs/quickstart_simple.md

```diff
@@ -67,37 +81,49 @@ cat > config.js
 Now open the config.js in your code editor and enter the following:
 
 ```Javascript
+require('dotenv').config()
+const HDWalletProvider = require("@truffle/hdwallet-provider");
+const fs = require("fs");
+const { homedir } = require('os');
 const { ConfigHelper } = require("@oceanprotocol/lib");
-const Web3 = require("web3");
-const defaultConfig = new ConfigHelper().getConfig("development");
 
-const urls = {
-  networkUrl: "http://localhost:8545",
-  aquarius: "http://localhost:5000",
-  providerUri: "http://localhost:8030",
-};
+var oceanConfig = new ConfigHelper().getConfig(process.env.OCEAN_NETWORK);
```

Comment:
either use `const/let` or the old `var`. But never mix them all in one file. So this should be a `const`